src/login/logind-action.h

```diff
@@ -21,12 +21,14 @@ typedef enum HandleAction {
 
 typedef struct ActionTableItem ActionTableItem;
 
-#define handle_action_valid(x) (x && (x < _HANDLE_ACTION_MAX))
-
 #include "logind-inhibit.h"
 #include "logind.h"
 #include "sleep-config.h"
 
+static inline bool handle_action_valid(HandleAction a) {
+        return a >= 0 && a < _HANDLE_ACTION_MAX;
```

Comment:
This change the behavior, previously, `a == 0` is not allowed. And I guess that causes the CI failure.

Reply:
the logic where the function is called is flawed. Was part of the refactoring, no idea how that happened. Sorry for that :-(
Fix:
https://github.com/systemd/systemd/pull/22522/commits/8c65875a9009b5f552893ef1f54ad2261c12f7ad

Reply:
now https://github.com/systemd/systemd/pull/22522/commits/54616cfa5c843b2d53ba2a36f50de98a94fefc37, should be cherry-pickable